### PR TITLE
Feat/cards

### PR DESCRIPTION
**Pull request by Debarshi95**
PR to add Card Component.
Fixes issue with current layout

### CODE REVIEW

File: lib/components/cards/index.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  height: 80%;
+  object-fit: fill;
+  width: 100%;
+  max-height: 14rem;
+}
+.card-content {
+  padding: 1rem;
+}
+.card-disabled {
+  background-color: #f9f9f9;
```
Comment (ShrutiLakhani) [inline lib/components/cards/index.css:20]: Please use color variables where ever possible

Comment (ShrutiLakhani) [inline lib/components/cards/index.css:20]: Change here as well

**Review by ShrutiLakhani — CHANGES_REQUESTED** [review]
Please make these changes and its good to merge

File: lib/components/cards/index.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
+.card {
+  max-width: 16rem;
+  height: fit-content;
+  max-height: 22rem;
+  box-shadow: 0 1px 4px 1px #acacac;
+  border-radius: 4px;
+  border: 1px solid #f9f9f9;
```
Comment (pankaj-prog) [inline lib/components/cards/index.css:7]: use color variables

File: lib/components/cards/index.css [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  object-fit: fill;
+  width: 100%;
+  max-height: 14rem;
+}
+.card-content {
+  padding: 1rem;
+}
+.card-disabled {
+  background-color: #f9f9f9;
+}
```
Comment (pankaj-prog) [inline lib/components/cards/index.css:21]: Good Naming

**Review by pankaj-prog — APPROVED** [review]

File: lib/components/cards/index.css (hunk 1 above)
Comment (Debarshi95) [reply lib/components/cards/index.css:20]: Will update that with variable.

File: lib/components/cards/index.css (hunk 2 above)
Comment (Debarshi95) [reply lib/components/cards/index.css:7]: Will update it with variables

**Review by ShrutiLakhani — COMMENTED** [review]
Clean UI. Looks good to merge. Good Job

**Pull request merged**